### ISSUE TITLE
Support queries using GraphQL

### ISSUE THREAD

**Issue by jrieken** [microsoft/vscode-github-issue-notebooks#18]
The v4-github-api supports GraphQL and it would make us a good sample for a polyglot notebook to support this.

**Comment by cognominal** [comment]
Indeed, beside being useful, the code helps to understand vscode API and github REST. The  query language is small so it helps to understand the mechanic of language features. So beside providing a programming sample,  supporting GraphQL may facilitate future contributions.
